File: src/main/java/pemilukm/teti/GlobalVar.java
package pemilukm.teti;

import javafx.event.EventHandler;
import javafx.scene.Scene;
import javafx.scene.input.KeyCombination;
import javafx.stage.Stage;
import javafx.stage.WindowEvent;

public class GlobalVar {
	public static Stage primaryStage;
    public static Scene mainScene;
    public static String valueNIM;
    public static String textNIM;
    public static boolean connActive;
    public static String AESKey = "<]pemilu!TETI19*";
    public static String passAdmin = "semangat";
    public static String passMain = "19lagi";

    public static void kiosk(){
        primaryStage.setAlwaysOnTop(true);
        //primaryStage.setMaximized(true);
        primaryStage.setFullScreen(true);
        primaryStage.setFullScreenExitHint(null);
        primaryStage.setFullScreenExitKeyCombination(KeyCombination.NO_MATCH);
        primaryStage.setOnCloseRequest(new EventHandler<WindowEvent>() {
            @Override
            public void handle(WindowEvent event) {
                event.consume();
            }
        });
    }
}

File: src/main/java/pemilukm/teti/GlobalVar.kt
package pemilukm.teti

import javafx.event.EventHandler
import javafx.scene.Scene
import javafx.scene.input.KeyCombination
import javafx.stage.Stage
import javafx.stage.WindowEvent

object GlobalVar {
    var primaryStage: Stage? = null
    var mainScene: Scene? = null
    var valueNIM: String? = null
    var textNIM: String? = null
    var connActive: Boolean = false
    var AESKey = "<]pemilu!TETI19*"
    var passAdmin = "semangat"
    var passMain = "19lagi"

    fun kiosk() {
        primaryStage!!.isAlwaysOnTop = true
        //primaryStage.setMaximized(true);
        primaryStage!!.isFullScreen = true
        primaryStage!!.fullScreenExitHint = null
        primaryStage!!.fullScreenExitKeyCombination = KeyCombination.NO_MATCH
        primaryStage!!.onCloseRequest = EventHandler { event -> event.consume() }
    }
}